doc/index.rst

```diff
@@ -70,6 +70,7 @@ in the community have done.
    customizing
    authentication
    known-deployments
+   federation/federation
```

Comment:
should lines 70, 71, 72, 73 have different indentations?

Reply:
do they? I can't see a difference

Reply:
I could swear they were indented by an extra character but not any more.